omnibus/config/software/inspec.rb

```diff
@@ -0,0 +1,49 @@
+# encoding: utf-8
+#
+# Copyright:: Copyright 2016, Chef Software Inc.
+# License:: Apache License, Version 2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+require_relative '../../../lib/inspec/version.rb'
+
+name 'inspec'
+
+dependency 'ruby'
+dependency 'bundler'
+dependency 'appbundler'
+
+license :project_license
+
+default_version "v#{Inspec::VERSION}"
+
+source path: "#{Omnibus::Config.project_root}/../"
```

Comment:
Need to exclude bundle/vendor if we end up setting up a Jenkins pipeline - the pipeline tends to do a bundle install in the omnibus directory and we don't want to pick up those files.